GUI/Main.Designer.cs

```diff
@@ -64,7 +65,7 @@ private void InitializeComponent()
             this.splitContainer1 = new System.Windows.Forms.SplitContainer();
             this.ModList = new CKAN.MainModListGUI();
             this.Installed = new System.Windows.Forms.DataGridViewCheckBoxColumn();
-            this.UpdateCol = new System.Windows.Forms.DataGridViewCheckBoxColumn();
+            this.Update = new System.Windows.Forms.DataGridViewCheckBoxColumn();
```

Comment:
OK, it looks like there were two declarations in the Designer file... I have had some issues mixing VS and the massive Main designer, will be more aware of them in the future, thanks.